winbuild/build.rst

```diff
@@ -42,8 +42,8 @@ behaviour of ``build_prepare.py``:
   If ``PYTHON`` is unset, the version of Python used to run
   ``build_prepare.py`` will be used. If only ``PYTHON`` is set,
   ``EXECUTABLE`` defaults to ``python.exe``.
-* ``ARCHITECTURE`` is used to select a ``x86`` or ``x64`` build. By default,
-  uses same architecture as the version of Python used to run ``build_prepare.py``.
+* ``ARCHITECTURE`` is used to select a ``x86``, ``x64`` or ``ARM64``build.
+  By default, uses same architecture as the version of Python used to run ``build_prepare.py``.
   is used.
```

Comment:
> From https://github.com/python-pillow/Pillow/pull/5811#issuecomment-961081944, win-arm64_pytests_full.txt:
> ```
> --- PIL CORE support ok, compiled for 9.0.0.dev0
> *** TKINTER support not installed
> *** FREETYPE2 support not installed
> *** LITTLECMS2 support not installed
> --- WEBP support ok, loaded 1.2.1
> --- WEBP Transparency support ok
> --- WEBPMUX support ok
> --- WEBP Animation support ok
> --- JPEG support ok, compiled for libjpeg-turbo 2.1.1
> --- OPENJPEG (JPEG2000) support ok, loaded 2.4.0
> --- ZLIB (PNG/ZIP) support ok, loaded 1.2.11
> --- LIBTIFF support ok, loaded 4.3.0
> *** RAQM (Bidirectional Text) support not installed
> *** LIBIMAGEQUANT (Quantization method) support not installed
> *** XCB (X protocol) support not installed
> ```

Based on this and the other PR you opened, it is my understanding that not all dependencies build successfully on ARM64. This should probably be noted in the readme, perhaps as follows:

```suggestion
  is used. Note that the ``ARM64`` build is not fully supported and some dependencies may fail to build. Pull requests are welcome.
```

Reply:
Yes, the current status not all optional external libraries are built.
However FreeType and libimagequant changes are upstreamed, I'm waiting for new releases to integrate, while littlecms2 is in review.

FreeType:
https://gitlab.freedesktop.org/freetype/freetype/-/commit/80bd4cba7eb25ecae313a6ac376ad6b363655243
https://gitlab.freedesktop.org/freetype/freetype/-/commit/e6e6cbf1648d4a776da0857921872f2fbc853205
https://github.com/mesonbuild/wrapdb/pull/205

libimagequant:
https://github.com/ImageOptim/libimagequant/commit/43c1c60e0302400b41ca9e69a2f0d2c710700647

littlecms2:
https://github.com/mm2/Little-CMS/pull/288